pkg/virt-handler/vm.go

```diff
@@ -817,6 +827,14 @@ func (d *VirtualMachineController) defaultExecute(key string,
 		log.Log.Info("Domain does not exist")
 	}
 
+	// make sure the per-vmi shared directory exists
+	interfaceCacheFile := "/var/run/kubevirt-network/%s/interface-cache-%s.json" // todo
```

Comment:
what's the `todo` comment here for?

Reply:
That's a remnant from when I was digging into why the directory is not visible to launcher pod. But that's now fixed, so this code is not needed here.

Reply:
Could you drop it then?

I also don't see you using the full path anywhere, could you stript the file name and make it interfaceCacheDirectory?